### ISSUE TITLE
[cli] Passing --no-prompt flag when installing apps and multi-cluster apps does not supply answers

### ISSUE THREAD

**Issue by rancher-max** [rancher/rancher#25514]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**
1. Do one of:
    1. `rancher app install openebs --no-prompt -n openebs`
    2. `rancher mcapp install openebs --role cluster-owner --no-prompt`
2. Run the corresponding inspect command:
    1. `rancher inspect --type app <id> | grep answers`
    2. `rancher inspect --type multiclusterapp <id> | grep answers`

**Result:**
Returns nothing back, but should return the object with answers present. 

**Other details that may be helpful:**
It looks like default answers are not actually being set when passing the `--no-prompt` flag. If updating the first command to not use `--no-prompt` and just hitting using all the defaults, the answers are set correctly.

**Environment information**
- Rancher v2.3.5
- Cli version downloaded from there:
```
$ rancher -v
rancher version v2.3.2
```
Also confirmed this is present with cli version: `v2.4.0-rc4`

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported):
- Machine type (cloud/VM/metal) and specifications (CPU/memory):
- Kubernetes version (use `kubectl version`):

```
$ rancher kubectl version
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.0", GitCommit:"70132b0f130acc0bed193d9ba59dd186f0e634cf", GitTreeState:"clean", BuildDate:"2019-12-07T21:20:10Z", GoVersion:"go1.13.4", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.2", GitCommit:"59603c6e503c87169aea6106f57b9f242f64df89", GitTreeState:"clean", BuildDate:"2020-01-18T23:22:30Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by brendarearden** [comment]
available for testing in rancher/cli v2.4.3-rc1

**Comment by rancher-max** [comment]
**I still see the same issue reported initially using v2.4.3-rc1 cli in rancher v2.4-head.**

I believe the problem is that there is no default "answers.yaml". The defaults are from "values.yaml" in the chart. I believe the UI sets an answers.yaml and passes that through to the API, which is "rancher flavor", whereas it appears the CLI might be using the standard "helm flavor" of values.yaml which is already present in the charts.

**Comment by kinarashah** [comment]
Available for testing in CLI version [v2.4.3-rc2](https://github.com/rancher/cli/releases/tag/v2.4.3-rc2)

**Comment by rancher-max** [comment]
Reopening because incorrect answers showing up in some cases. This is noticeable by following the steps mentioned in this issue, and then comparing it to the same but without using `--no-prompt` flag. The values that appear in the "answers" field are different.

**Comment by kinarashah** [comment]
Available to test in `v2.4.3-rc3`[https://github.com/rancher/cli/releases/tag/v2.4.3-rc3](url)

**Comment by rancher-max** [comment]
#### Validated in v2.4.3-rc3

Performed steps from original issue and validated it is working now. The command `rancher inspect --type app <id> | grep answers` still does not retrieve anything, but there are the correct values when grepping `valuesYaml`, so this is actually OK as it is a design decision.

Example what the answers look like when viewed through UI after deploying through CLI with --no-prompt:
<img width="1260" alt="image" src="https://user-images.githubusercontent.com/55997940/80264504-3cdbe500-8649-11ea-9bc8-b43f09a78b23.png">

#### Reopening because multi-cluster apps are still failing

**Comment by kinarashah** [comment]
Available to test in [v2.4.3-rc4](https://github.com/rancher/cli/releases/tag/v2.4.3-rc4)

**Comment by rancher-max** [comment]
#### Validated in cli version v2.4.3-rc4 using rancher v2.4-head

- Mcapps and Apps both work when doing the steps noted in the original issue comment. 
- All reasons for reopening have been resolved.
- General CLI regression passes
